client/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "env": {
+    "amd": true,
```

Comment:
Actually, sorry, we don't need `"amd": true` here since it is already in the parent .eslintrc file.